src/site/stages/build/plugins/create-drupal-debug.js

```diff
@@ -33,7 +33,8 @@ function createDrupalDebugPage(buildOptions) {
   // a debug page should never be built there.
   if (
     !ENABLED_ENVIRONMENTS.has(buildOptions.buildtype) ||
-    buildOptions.buildtype === ENVIRONMENTS.VAGOVPROD
+    buildOptions.buildtype === ENVIRONMENTS.VAGOVPROD ||
+    buildOptions.omitdebug === true
```

Comment:
I can’t imagine this plugin is useful for anyone anymore. I think we should just delete this whole file.